run_linux.go

```diff
@@ -2381,6 +2391,54 @@ func runSetupRunMounts(mounts []string, secrets map[string]string, sshSources ma
 	return finalMounts, artifacts, nil
 }
 
+func getBindMount(builderInstance *Builder, tokens []string, rootUID, rootGID, processUID, processGID int) (*spec.Mount, error) {
+	errInvalidSyntax := errors.New("--mount should have syntax source=path[optional],target=path")
+	if len(tokens) == 0 {
+		return nil, errInvalidSyntax
+	}
+	var source, target, mountOpts string
+	var volumeMounts []string
+
+	for _, val := range tokens {
+		kv := strings.SplitN(val, "=", 2)
+		switch kv[0] {
+		case "source":
+			source = kv[1]
+		case "target", "dst", "destination":
+			target = kv[1]
+		default:
+			return nil, errInvalidSyntax
+		}
+	}
+	if source == "" || source == "." {
```

Comment:
is the '.' check necessary?  Wouldn't that resolve?  If it doesn't resolve, do you need a check for `./` too?

Reply:
resolved